src/libraries/System.Private.Xml/src/System/Xml/Serialization/Mappings.cs

```diff
@@ -1094,7 +1104,7 @@ internal SerializableMapping(XmlQualifiedName xsiType, XmlSchemaSet schemas)
             _needSchema = false;
         }
 
-        internal void SetBaseMapping(SerializableMapping mapping)
+        internal void SetBaseMapping(SerializableMapping? mapping)
```

Comment:
I think `mapping` should not be `nullable` here? Otherwise https://github.com/dotnet/runtime/pull/41261/files#diff-81667efb15f99a82497ac6bfe0ae7ea5R492 will be a NRE?

Reply:
Hmm that link didn't lead me anywhere.

Reply:
Ah, bad link. Try this instead: https://github.com/dotnet/runtime/pull/41261/files?file-filters%5B%5D=#diff-81667efb15f99a82497ac6bfe0ae7ea5R492. Although, now when I see it again, it is not a bug, so this change is good.